packages/react-instantsearch-core/src/core/createInstantSearchManager.js

```diff
@@ -1,8 +1,20 @@
-import { omit, find } from 'lodash';
-import algoliasearchHelper, { SearchParameters } from 'algoliasearch-helper';
+import { omit } from 'lodash';
+import algoliasearchHelper from 'algoliasearch-helper';
 import createWidgetsManager from './createWidgetsManager';
 import createStore from './createStore';
 import { HIGHLIGHT_TAGS } from './highlight';
+import { hasMultipleIndex } from './indexUtils';
```

Comment:
could be called `hasMultipleIndices`

Reply:
Will do it in another PR this function is used in multiple places.

Reply:
Fixed inside https://github.com/algolia/react-instantsearch/pull/1852